CONTRIBUTING.md

```diff
@@ -2,24 +2,38 @@ Thanks for contributing!
 
 ## Development
 
-Run `npm run examples` to run a webpack dev server with Radium examples.
+### Installing dependencies
 
-## Lint
+```bash
+npm install
+```
 
-Run `npm run lint` before committing to lint files.
+You will find all building blocks that make up Radium in the [`modules`](modules) folder.
 
-## Flow
+### Testing
 
-1. [Install flow](http://flowtype.org/docs/getting-started.html#installing-flow)
-2. Run `flow` to check for missing annotations and static type check failures
+You will find tests for each module inside [`modules/__tests__`](modules/__tests__). Whenever making any changes, ensure that all existing tests pass by running `karma start`.
+
+If you are adding a new feature or some extra functionality, you should also make sure to accompany those changes with appropriate tests.
+
+### Linting
+
+Before commiting any changes, be sure to do `npm run lint`; this will lint all relevant files using [ESLint](http://eslint.org/) and report on any changes that you need to make.
+
+### Flow
+
+- [Install flow](http://flowtype.org/docs/getting-started.html#installing-flow)
+- Run `flow` to check for missing annotations and static type check failures
+
+### Examples
+
+You can run examples locally by simply doing `npm run examples`; this will start a webpack dev server (the config file is found at [examples/webpack.config.js](examples/webpack.config.js)) making Radium examples available at `http://localhost:8080`.
```

Comment:
Note that if you use `npm run test-dev` as above, karma will use port 8080. You can change the port used by the examples by running `npm run examples -- --port 8000`.